spring-cloud-stream-binder-kinesis/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-stream-binder-kinesis</artifactId>
+	<packaging>jar</packaging>
+	<name>spring-cloud-stream-binder-kinesis</name>
+	<description>Kinesis binder implementation</description>
+
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-stream-binder-kinesis-parent</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+
+	<properties>
+		<spring-cloud-stream.version>1.2.0.BUILD-SNAPSHOT</spring-cloud-stream.version>
+		<spring-integration-aws-version>1.1.0.M1</spring-integration-aws-version>
+		<aws-sdk-version>1.11.128</aws-sdk-version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-binder-kinesis-core</artifactId>
+			<version>1.0-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream</artifactId>
```

Comment:
I think we don't need this here because it should be transitive from the `spring-cloud-stream-binder-kinesis-core` anyway.
The same for the `aws-java-sdk-kinesis`